It looks like dependency-management/pom-type is failing. Maybe this is one of tests that's resolving from an Ivy layout repository.

```scala
[info] Running dependency-management/pom-type
[info] [info] welcome to sbt 2.0.0-RC9-bin-SNAPSHOT (Azul Systems, Inc. Java 25.0.2)
[info] [info] loading global plugins from /tmp/sbt_8b3b2db0/global/plugins
[error] -- Warning: /tmp/sbt_8b3b2db0/build.sbt:18:49 ----------------------------------
[error] 18 |      val hasError = lines exists { line => line contains "Found intransitive dependency "}
[error]    |                                                 ^^^^^^^^
[error]    |Alphanumeric method contains is not declared infix; it should not be used as infix operator.
[error]    |Instead, use method syntax .contains(...) or backticked identifier `contains`.
[error]    |The latter can be rewritten automatically under -rewrite -source 3.4-migration.
[info] [info] set current project to root (in build file:/tmp/sbt_8b3b2db0/)
[info] [info] Wrote /tmp/sbt_8b3b2db0/target/out/jvm/scala-2.10.6/root/root_2.10-0.1.0-SNAPSHOT.pom
[info] [error] lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
[info] [error] https://repo.typesafe.com/typesafe/ivy-releases/org.scala-tools.sbinary/sbinary_2.10/0.4.1/jars/sbinary.jar: not found: https://repo.typesafe.com/typesafe/ivy-releases/org.scala-tools.sbinary/sbinary_2.10/0.4.1/jars/sbinary.jar
[info] [error] 
[info] [error] 	at lmcoursier.internal.shaded.coursier.Artifacts$.$anonfun$fetchArtifacts$9(Artifacts.scala:365)
[info] [error] 	at lmcoursier.internal.shaded.coursier.util.Task$.$anonfun$flatMap$extension$1(Task.scala:14)
[info] [error] 	at lmcoursier.internal.shaded.coursier.util.Task$.$anonfun$flatMap$extension$1$adapted(Task.scala:14)
[info] [error] 	at lmcoursier.internal.shaded.coursier.util.Task$.wrap(Task.scala:82)
[info] [error] 	at lmcoursier.internal.shaded.coursier.util.Task$.$anonfun$flatMap$2(Task.scala:14)
[info] [error] 	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:506)
[info] [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
[info] [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
[info] [error] 	at java.base/java.lang.Thread.run(Thread.java:1474)
[info] [error] Caused by: lmcoursier.internal.shaded.coursier.cache.ArtifactError$NotFound: not found: https://repo.typesafe.com/typesafe/ivy-releases/org.scala-tools.sbinary/sbinary_2.10/0.4.1/jars/sbinary.jar
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader$Blocking$.doDownload(Downloader.scala:239)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader$Blocking$.$anonfun$remote$2(Downloader.scala:423)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.loop$1(CacheLocks.scala:94)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withLockOr(CacheLocks.scala:117)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader$Blocking$.$anonfun$remote$1(Downloader.scala:426)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader$.$anonfun$downloading$2(Downloader.scala:776)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.CacheLocks$.withUrlLock(CacheLocks.scala:136)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader$.$anonfun$downloading$1(Downloader.scala:776)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Retry.loop$1(Retry.scala:23)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Retry.retryOpt(Retry.scala:39)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader$.coursier$cache$internal$Downloader$$downloading(Downloader.scala:812)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader$Blocking$.remote(Downloader.scala:428)
[info] [error] 	at lmcoursier.internal.shaded.coursier.cache.internal.Downloader.$anonfun$remote$6(Downloader.scala:546)
[info] [error] 	at scala.concurrent.Future$.apply$$anonfun$1(Future.scala:692)
[info] [error] 	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:503)
[info] [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
[info] [error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
[info] [error] 	at java.base/java.lang.Thread.run(Thread.java:1474)
[info] [error] (update) lmcoursier.internal.shaded.coursier.error.FetchError$DownloadingArtifacts: Error fetching artifacts:
[info] [error] https://repo.typesafe.com/typesafe/ivy-releases/org.scala-tools.sbinary/sbinary_2.10/0.4.1/jars/sbinary.jar: not found: https://repo.typesafe.com/typesafe/ivy-releases/org.scala-tools.sbinary/sbinary_2.10/0.4.1/jars/sbinary.jar
[info] [error] elapsed time: 1 s, cache 100%, 2 disk cache hits
[error] x dependency-management/pom-type
```

--- a/sbt-app/src/sbt-test/dependency-management/pom-type/build.sbt
+++ b/sbt-app/src/sbt-test/dependency-management/pom-type/build.sbt
@@ -2,9 +2,9 @@ lazy val checkPom = taskKey[Unit]("check pom to ensure no <type> sections are ge
 
 lazy val root = (project in file(".")).
   settings(
-    scalaVersion := "2.10.6",
-    libraryDependencies += { ("org.scala-tools.sbinary" %% "sbinary" % "0.4.1").withSources().withJavadoc() },
-    libraryDependencies += { ("org.scala-sbt" % "io" % "0.13.8").intransitive() },
+    scalaVersion := "2.13.16",
+    libraryDependencies += { ("com.typesafe" % "config" % "1.4.3").withSources().withJavadoc() },
+    libraryDependencies += { ("org.slf4j" % "slf4j-api" % "2.0.16").intransitive() },
     checkPom := {
       val converter = fileConverter.value
       val pomFile = makePom.value
@@ -16,8 +16,7 @@ lazy val root = (project in file(".")).
       val ur = update.value
       val dir = (update / streams).value.cacheDirectory / "out"
       val lines = IO.readLines(dir)
-      val hasError = lines exists { line => line contains "Found intransitive dependency "}
+      val hasError = lines.exists(line => line.contains("Found intransitive dependency "))
       assert(hasError, s"Failed to detect intransitive dependencies, got: ${lines.mkString("\n")}")
     },
-    resolvers += Resolver.typesafeIvyRepo("releases")
   )

